tests/test_scripts.py

```diff
@@ -529,6 +529,23 @@ def test_combine_outfile(tmpdir, script_runner):
     assert len(combined_ws.measurement_names) == 8
 
 
+def test_combine_merge_channels(tmpdir, script_runner):
+    temp_1 = tmpdir.join("parsed_output.json")
+    temp_2 = tmpdir.join("renamed_output.json")
+    command = f'pyhf xml2json validation/xmlimport_input/config/example.xml --basedir validation/xmlimport_input/ --output-file {temp_1.strpath} --hide-progress'
+    ret = script_runner.run(*shlex.split(command))
+
+    command = (
+        f'pyhf prune {temp_1.strpath} --sample signal --output-file {temp_2.strpath}'
+    )
+
+    ret = script_runner.run(*shlex.split(command))
```

Comment:
```suggestion
    ret = script_runner.run(*shlex.split(command))
    assert ret.success

    command = (
        f'pyhf prune {temp_1.strpath} --sample signal --output-file {temp_2.strpath}'
    )

    ret = script_runner.run(*shlex.split(command))
    assert ret.success
```

For clarity we should probably not not assign if things are never going to be used. Can we either remove the assignment to `ret` or add in asserts?